src/opera/commands/deploy.py

```diff
@@ -57,3 +73,30 @@ def deploy(args):
         return 1
 
     return 0
+
+
+def deploy_compressed_csar(args, storage):
+    with TemporaryDirectory(dir=storage.path) as tempdir:
+        ZipFile(args.csar.name, 'r').extractall(tempdir)
+
+        try:
+            inputs_file = open(tempdir + "/" + args.inputs, 'r')
+            inputs = yaml.safe_load(inputs_file) if args.inputs else {}
+            storage.write_json(inputs, "inputs")
+        except Exception as e:
+            print("Invalid inputs: {}".format(e))
+            return 1
+
+        try:
+            ast = tosca.load(Path.cwd(), PurePath(tempdir + "/" + args.template))
```

Comment:
Since we're using pathlib, let's use it all the way and not mix string concatenation.

Reply:
This should be fixed now.